### ISSUE TITLE
[Bug]: It show error on device manager (code 43)

### ISSUE THREAD

**Issue by Fleishmann83** [itsmikethetech/Virtual-Display-Driver#189]
### Which OS?

Win11 Pro

### Which release?

24H2

### Describe the bug

After working great for like 3 hours, and several reboots (the last thing i did was setting up autologin on W11 with netplzwiz), it shows the yellow triangle on device manager and stop working)
Reinstalling didn't work 

### Contact Details

krilin2012@hotmail.com

**Comment by Fleishmann83** [comment]
I tried with the no-HDR version and it's the same. It seems to be OK, but shows code 43 after reboot.

**Comment by itsmikethetech** [comment]
Did you happen to delete the driver folder after installation?

**Comment by itsmikethetech** [comment]
If you disable and reenable in Device Manager, does the code persist?

**Comment by Fleishmann83** [comment]
It shows no error until I reboot or I try to use it (like starting a stream in sunshine)

**Comment by bud3699** [comment]
Do you have multiple accounts on windows ?

**Comment by Fleishmann83** [comment]
No, but the last thing before troubles was remove the windows 11 PIN and using netlplwiz to auto-enter the user password, with the purpose of enabling wake on lan for sunshine.

I think that in the next reboot then problems began.

**Comment by Fleishmann83** [comment]
Update

If I disable autologin and set PIN login again, the driver works as intended. 

The problem here is to set my remote tun on and login procedure.

**Comment by zjoasan** [comment]
@NanashiTheNameless Am I  understanding you correctly, that use also use a "auto login" system?

This might be a hard but to crack in the current driver model, I have few ideas but need to takk to Mike.

1. User/kernel mode problem (likely due to altered functionality on auto login) 

2. The how AutoLogin is set up

Will need more details, driver warning, Windows version(detailed, both updates etc), Version of VDD, setting. XML. And any other info you think might have changed the behavior.

Thanks / Jocke

**Comment by NanashiTheNameless** [comment]
> [@NanashiTheNameless](https://github.com/NanashiTheNameless) Am I understanding you correctly, that use also use a "auto login" system?
> 
> This might be a hard but to crack in the current driver model, I have few ideas but need to takk to Mike.
> 
>     1. User/kernel mode problem (likely due to altered functionality on auto login)
> 
>     2. The how AutoLogin is set up
> 
> 
> Will need more details, driver warning, Windows version(detailed, both updates etc), Version of VDD, setting. XML. And any other info you think might have changed the behavior.
> 
> Thanks / Jocke



https://learn.microsoft.com/en-us/sysinternals/downloads/autologon

https://valid.x86.fr/mbpef0

```xml
<?xml version="1.0" encoding="utf-8"?>
<vdd_settings>
  <monitors>
    <count>1</count>
  </monitors>
  <gpu>
    <friendlyname>AMD Radeon RX 7900 XT</friendlyname>
  </gpu>
  <global>
    <g_refresh_rate>60</g_refresh_rate>
    <g_refresh_rate>90</g_refresh_rate>
    <g_refresh_rate>120</g_refresh_rate>
    <g_refresh_rate>144</g_refresh_rate>
    <g_refresh_rate>165</g_refresh_rate>
    <g_refresh_rate>244</g_refresh_rate>
  </global>
  <resolutions>
    <resolution>
      <width>2560</width>
      <height>1440</height>
      <refresh_rate>60</refresh_rate>
    </resolution>
  </resolutions>
  <options>
    <CustomEdid>false</CustomEdid>
    <PreventSpoof>false</PreventSpoof>
    <EdidCeaOverride>false</EdidCeaOverride>
    <HardwareCursor>true</HardwareCursor>
    <SDR10bit>true</SDR10bit>
    <HDRPlus>false</HDRPlus>
    <logging>false</logging>
    <debuglogging>false</debuglogging>
  </options>
</vdd_settings>
```
